packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -935,7 +935,15 @@ function isOverTarget(point: EventPoint, target: Element) {
 
 function shouldPreventDefaultDown(target: Element) {
   // We cannot prevent default if the target is a draggable element.
-  return !(target instanceof HTMLElement) || !target.hasAttribute('draggable');
+  if (!(target instanceof HTMLElement)) {
+    return true;
+  }
+
+  if (target.tagName === 'A' && target.getAttribute('draggable') !== 'false') {
+    return false;
+  }
+
+  return !target.hasAttribute('draggable');
```

Comment:
Q: I think even if target element has `draggable="false"`, `target.hasAttribute('draggable')` is `true` (because `hasAttribute` ignore its value), is it ok? or should we fix it as `!(target.getsAttribute('draggable') === "true");`